v22.2/inverted-indexes.md

```diff
@@ -61,19 +61,19 @@ This lets you search based on subcomponents.
 
 You can use GIN indexes to improve the performance of queries using `JSONB` or `ARRAY` columns. You can create them:
 
-- At the same time as the table with the `INVERTED INDEX` clause of [`CREATE TABLE`](create-table.html#create-a-table-with-secondary-and-gin-indexes).
 - For existing tables with [`CREATE INDEX`](create-index.html):
```

Comment:
At the end of the first bullet point here (after `CREATE INDEX`) there is a colon, assume that needs to change?